### ISSUE TITLE
Allow Unicode symbols for list items such as •, ◦, ‣ 

### ISSUE THREAD

**Issue by toastal** [jgm/djot#195]
Unicode is nice and typography and word processors have some established [bullet symbols](https://en.wikipedia.org/wiki/Bullet_(typography)). I find them to be a more pleasant experience to read plaintext document with these symbols over `*` or `-` as they have many different meanings to me due to ANSI compatibility.

**Comment by vassudanagunta** [comment]
Yes, they avoid the ambiguity of `-`, though djot avoids this by requiring a blank line before a new list.

**Comment by toastal** [comment]
It's mostly just an aesthetic choice, but has some history to it. With a pen we would certainly write `•`, but keyboard limitations are the reason we go for `-` and `*`. However in 2023 we certainly have the ability to type `•` (with the Vim digraph `oo` supported since 2004).

**Comment by bpj** [comment]
There is some difficulty in that Lua can't match chars above U+007F as classes. You will have to either match *any* character and look it up in a list, or try one match for (the bytes of) each possible char, both of which are suboptimal to say the least. If you want to preserve Lua compatibility that is.
